docs/kernel-policy.md

```diff
@@ -1,59 +1,25 @@
 # Firecracker's Kernel Support Policy
 
-Being a virtual machine monitor, Firecracker represents a component in a
-larger stack, one in which it is tightly coupled with the guest and host
-kernels on which it is run. The presented kernel support policy aims at
-offering customers predictability into future kernel related changes.
-
-As of right now, Firecracker code from main branch is being validated
-continuously on *4.14* and *5.10* host and guest kernels. While other
-versions and other kernel configs might work, they are not periodically
-validated in our test suite, and using them might result in unexpected behaviour.
-Once enabled, a kernel version is supported for a **minimum of 2 years**.
+Once enabled, kernel versions are supported for a **minimum of 2 years**.
 
 We are validating the currently supported Firecracker releases as per
-[Firecracker’s release policy](../docs/RELEASE_POLICY.md).
-Starting with release `v1.0` each major and minor release will specify
-the supported kernel versions. Adding support for a new kernel version
-will result in a Firecracker release only if compatibility changes are
-required.
-
-The currently supported kernel versions can be seen in the table below.
-Based on our user requests, every year we are considering for enablement
-the latest LTS version.
-
-<table>
-  <tr>
-    <th></th>
-    <th>2022</th>
-  </tr>
-  <tr>
-    <td>4.14</td>
-    <td style="background-color:mediumseagreen">full support</td>
-  </tr>
-  <tr>
-    <td>5.10</td>
-    <td style="background-color:mediumseagreen">full support</td>
-  </tr>
-</table>
+[Firecracker’s release policy](../docs/RELEASE_POLICY.md). Starting with release
+`v1.0` each major and minor release will specify the supported kernel versions.
+Adding support for a new kernel version will result in a Firecracker release
+only if compatibility changes are required.
 
 The guest kernel configs used in our validation pipelines
 can be found [here](../resources/guest_configs/) while a breakdown
 of the relevant guest kernel modules can be found in the next section.
 
-**Note**
-Be sure to check feature compatibility between host, guest kernels
-and the hardware.
-For the combination of 4.14 host and 5.10 guest kernels we are using
-different configuration of the guest kernel with SVE extension disabled.
-This is due to the introduction of the SVE extension in Graviton3,
-which causes the default 5.10 guest kernel, which has SVE support,
-to crash if run on top of the host with 4.14 kernel which does not support SVE.
+When using a 4.14 host and a 5.10 guest, we disable the SVE extension in the
```

Comment:
Maybe this should be in another section "Known caveats"

Reply:
It would give it prominence.

You could put a section at the end with `# Caveats` and also the snapshot versioning caveats. Potentially moving those to `snapshotting/snapshot-support.md`  and just a reference that.

Reply:
Added a `Caveats` section which includes this and `Snapshot compatibility across kernel versions` which has been moved.